src/components/Application-Body/header/Header.tsx

```diff
@@ -15,24 +17,27 @@ import PostNewJob from './PostNewJob/PostNewJob';
 
 import { getAllSelectOptions } from '../../../server/ads';
 import { AllSelectOptions } from '../../../types/AllSelectOptions';
+import { UserStore } from '../../../store/UserStore';
 
 interface HeaderOwnProps {
     fetchAllAdsAfterPost: () => void;
     searchValue: string;
     onSearchValueChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    userStore?: UserStore;
```

Comment:
Why do we have `?`, if we do in line 32 `!`, where we assume it always exists? 🤔